test/handleGithubComment.test.js

```diff
@@ -0,0 +1,197 @@
+          "timeline_url": "https://api.github.com/repos/Automattic/mongoose/issues/11038/timeline",
+          "performed_via_github_app": null
+        },
+        "comment": {
+          "url": "https://api.github.com/repos/Automattic/mongoose/issues/comments/994888151",
+          "html_url": "https://github.com/Automattic/mongoose/issues/11038#issuecomment-994888151",
+          "issue_url": "https://api.github.com/repos/Automattic/mongoose/issues/11038",
+          "id": {
+            "$numberInt": "994888151"
+          },
+          "node_id": "IC_kwDOAAkfd847TMnX",
+          "user": {
+            "login": "vkarpov15",
+            "id": {
+              "$numberInt": "1620265"
+            },
+            "node_id": "MDQ6VXNlcjE2MjAyNjU=",
+            "avatar_url": "https://avatars.githubusercontent.com/u/1620265?v=4",
+            "gravatar_id": "",
+            "url": "https://api.github.com/users/vkarpov15",
+            "html_url": "https://github.com/vkarpov15",
+            "followers_url": "https://api.github.com/users/vkarpov15/followers",
+            "following_url": "https://api.github.com/users/vkarpov15/following{/other_user}",
+            "gists_url": "https://api.github.com/users/vkarpov15/gists{/gist_id}",
+            "starred_url": "https://api.github.com/users/vkarpov15/starred{/owner}{/repo}",
+            "subscriptions_url": "https://api.github.com/users/vkarpov15/subscriptions",
+            "organizations_url": "https://api.github.com/users/vkarpov15/orgs",
+            "repos_url": "https://api.github.com/users/vkarpov15/repos",
+            "events_url": "https://api.github.com/users/vkarpov15/events{/privacy}",
+            "received_events_url": "https://api.github.com/users/vkarpov15/received_events",
+            "type": "User",
+            "site_admin": false
+          },
+          "created_at": "2021-12-15T15:13:42Z",
+          "updated_at": "2021-12-15T15:13:42Z",
+          "author_association": "COLLABORATOR",
+          "body": "This is expected behavior in 6.x, we removed the `omitUndefined` option and made it true always. That means Mongoose will always strip undefined keys from updates. There's an entry in the [changelog](https://github.com/Automattic/mongoose/blob/master/CHANGELOG.md#600-rc0--2021-08-03), and we'll add a note to the migration guide with #10672. ",
+        },
+      };
+
+      const stub = sinon.stub(slack, 'sendMessage').callsFake(() => Promise.resolve('Message Sent!'));
+
+      await handleGithubComment({task, conn})(params);
+      assert(stub.callCount);
+      assert(stub.returns('Message Sent!'))
```

Comment:
This assertion isn't useful because you're asserting something that was defined in the test.

Better approach: test what the [sinon stub was called with](https://masteringjs.io/tutorials/sinon/stub-called-with). You don't need the whole Slack payload, but add an assertion that `stub` was called with an object that has `channel === '#pro-notifications'` and `blocks.text.text` contains 'vkarpov15'